client_cpp/src/posts/body.cc

```diff
@@ -77,12 +77,24 @@ Node Post::render_body()
         if (state.italic) {
             state.append({ "i" }, true);
         }
+        if (state.red) {
+            state.append({ "span style=\"color:red;\"" }, true);
```

Comment:
The params should be specified as `({"span", {"style", "color:red;"}}, true)`. See `brunhild/node.hh:Node`. 

I'd still prefer, if this was configurable for each theme with classes. Actually, you can reuse the admin and mod colours. Just use the `admin` class for red and `mod` class for blue. You'll need to add 
```
.mod {
	color: @mod;
}
```
to `less/common.mix.less`.

So that would be `({"span", {"class", "admin"}}, true)` and so on.

Reply:
I'll use them, but I'm going to have to update the ocean and gar themes to match.